html/Corporate/index.html

```diff
@@ -0,0 +1,349 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Corporate</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <!-- Banner -->
+  <section class="banner">
+    <div class="banner__container">
+      <!-- Title -->
+      <header class="banner__title">
+        There is no other platforms for you as like 
+      </header>
+
+      <!-- Play -->
+      <button type="button" class="banner__play">
+        <img  src="./assets/play-icon/play-video-icon.svg" alt="">
+      </button> 
+
+      <!-- Button -->
+      <button class="banner__try-now" type="button">Try now</button>
```

Comment:
i would suggest do not name classes by content, try to use something like "action", "interaction", "button", etc